lms/templates/forgot_password_modal.html

```diff
@@ -49,10 +51,17 @@ <h2>${_("Password Reset")}</h2>
   (function() {
     $(document).delegate('#pwd_reset_form', 'ajax:success', function(data, json, xhr) {
       if(json.success) {
-        $("#password-reset").html(json.value);
+        $("#password-reset").html(edx.HtmlUtils.ensureHtml(json.value).toString());
```

Comment:
The way `ensureHtml` works, is that it needs an `HtmlSnippet` to avoid escaping, which would only be created by another `HtmlUtils` call, like `interpolateHtml` below as an example.  Otherwise, it will just HTML-escape.

If `json.value` is plain-text, just use `$("#password-reset").text(json.value)`.  If it returns HTML, then you should should ensure it is bleached before returned from the view, and here you could use `edx.HtmlUtils.HTML(...)` with a comment like `// HTML was bleached server-side`.

Reminder to manually test this.

Reply:
Ok it seems this file is no more in use. This is being included in login.html https://github.com/edx/edx-platform/blob/master/lms/templates/login.html#L147 and login.html (https://github.com/edx/edx-platform/blob/master/openedx/core/djangoapps/user_authn/views/deprecated.py#L56) which seems to be deprecated by now. I think the new page login/registration page is in JS backbone altogether. So for now iam making it ```$("#password-reset").text(json.value)``` and let generate any issue and we'll forward fix.

Reply:
@adeelkhan: That seems fine.
@nasthagiri: Do you have info on this older page?